server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -93,23 +87,19 @@ public StatusResource(RulesCurator rulesCurator, Configuration config, JsRunnerP
             standalone = false;
         }
 
-        try (FileReader reader = new FileReader("/etc/candlepin/keycloak.json")) {
+        try (FileReader reader = new FileReader(ConfigProperties.KEYCLOAK_FILEPATH)) {
             //Read JSON file
-            JSONParser jsonParser = new JSONParser();
-            JSONObject jsonobj = (JSONObject) jsonParser.parse(reader);
-            realm = (String) jsonobj.get("realm");
-            authUrl = (String) jsonobj.get("auth-server-url");
-            resource = (String) jsonobj.get("resource");
-
+            ObjectMapper objectMapper = new ObjectMapper();
+            JsonNode rootNode = objectMapper.readTree(reader);
```

Comment:
A couple important things here:

1. We don't want to roll our own parser if we don't need to. Keycloak already provides this in the form of AdapterConfig.
2. We should not be performing configuration parsing/loading at this point. It should already have been done and we should be referencing that already-parsed config at this point.